PSKoans/Koans/dbatools/AboutDbaDatabase.ps1

```diff
@@ -0,0 +1,88 @@
+#Requires -Modules dbatools
+using module PSKoans
+[Koan(Position = 1001)]
+param()
+<#
+    Get-DbaDatabase
+
+    The Get-DbaDatabase command gets SQL database information for each database
+    that is present on the target instance(s) of SQL Server by default. If the 
+    name of the database is provided, the command will return only the specific 
+    database.
+#>
+Describe 'Get-DbaDatabase' {
+
+    <#
+        Let's setup the environment for you. Unless you want your Koans to 
+        nearly always fail I'd suggest not messing with this bit.
+    #>
+    Mock -CommandName Get-DbaDatabase -MockWith {
```

Comment:
I'd recommend adding `#region Mocks` / `#endregion` comment markers before and after these kinds of things when you need these. That way most editors will let folks fold them away completely and not have to worry about accidentally touching them.

You will also need to put them inside a `BeforeAll { }` block as well.